src/compas/geometry/curves/polyline.py

```diff
@@ -6,11 +6,11 @@
 
 from compas.geometry import allclose
 from compas.geometry import transform_points
-from compas.geometry import is_point_on_line
+from compas.geometry import is_point_on_line, is_point_on_polyline
 from compas.geometry import Point
 from compas.geometry import Line
 from compas.geometry import Frame
-from .curve import Curve
+from compas.geometry.curves.curve import Curve
```

Comment:
why change this?

Reply:
I think that at the time I was getting an import error, now that I tried again everything works well. I probably forgot to change it to the original when I was done